lib/galaxy/managers/configuration.py

```diff
@@ -56,6 +57,13 @@ def decode_id(
         decoded_id = self._app.security.decode_id(encoded_id)
         return {"decoded_id": decoded_id}
 
+    def encode_id(
+        self,
+        decoded_id: int,
+    ) -> Dict[str, str]:
+        encoded_id = unicodify(self._app.security.encode_id(decoded_id))
```

Comment:
The result of `self._app.security.encode_id` will be already "unicodified" so you can safely remove this call and the import.

https://github.com/galaxyproject/galaxy/blob/566731bdb0f80d45aa3d669b2a6fd0024af8fb78/lib/galaxy/security/idencoding.py#L49

```suggestion
        encoded_id = self._app.security.encode_id(decoded_id)
```